tests/testthat/test_spatial_deviation_noise_matrix_rng.R

```diff
@@ -0,0 +1,73 @@
+test_that(
+  "Test if non-centered parameter transformation of MVN(0,I) agrees with stan
+  spatial_deviation_noise_matrix_rng.stan",
+  {
+    model <- compiled_site_inf_model
+    space_model_fxns <- spatial_fxns$functions
+
+    corr_fun_params <- list(
+      dist_matrix = as.matrix(
+        dist(
+          data.frame(
+            x = c(85, 37, 48, 7),
+            y = c(12, 75, 81, 96),
+            diag = TRUE,
+            upper = TRUE
+          )
+        )
+      ),
+      phi = 25,
+      l = 1
+    )
+    omega_matrix <- space_model_fxns$exponential_decay_corr_func(
+      corr_fun_params$dist_matrix,
+      corr_fun_params$phi,
+      corr_fun_params$l
+    )
+    sigma_matrix <- sqrt(0.02) * omega_matrix
```

Comment:
I would define `sigma_eps = sqrt(0.02)` for readability. 

Also, just confirming that this doesn't include any normalizing by the maximum distance or ensuring that the trace of $$\Sigma$$ is equal to the number of sites?